### ISSUE TITLE
Syntax error when trying to export data (json, csv, xml)

### ISSUE THREAD

**Issue by kblomqvist** [sosedoff/pgweb#203]
In SQL Query tab whenever I press JSON, CSV or XML, I'll get error

```
{"error":"pq: syntax error at or near \"U0VMRUNUICogRlJPTSB3ZWF0aGVyOw\""}
```

It doesn't matter whether I have made a query, e.g. `SELECT * FROM foo`.

**Comment by sosedoff** [comment]
It should be already fixed in #186 and available in master. Try that.
New pgweb version (with fixes) is not released yet.

**Comment by kblomqvist** [comment]
Updated to the latest release. Works fine now.
